src/loaders/tvg/tvgTvgLoadParser.cpp

```diff
@@ -20,76 +20,85 @@
  * SOFTWARE.
  */
 
-#include <string.h>
+#include <memory.h>
 #include "tvgTvgLoadParser.h"
 
+
+/************************************************************************/
+/* Internal Class Implementation                                        */
+/************************************************************************/
+
 enum class LoaderResult { InvalidType, Success, SizeCorruption, MemoryCorruption, LogicalCorruption };
 
 static LoaderResult _parsePaint(tvgBlock block, Paint ** paint);
 
-static tvgBlock _readTvgBlock(const char *pointer)
+static tvgBlock _readTvgBlock(const char *ptr)
 {
```

Comment:
I prefer more descriptive names. what do you think about ptr -> bufferPosition ? or is it too long?

Reply:
pointer / ptr is enough to understand. just make it reduce.

Reply:
ok, I'll change the name in the saver